collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -354,7 +354,7 @@ public void onClick(View v) {
 							instanceCursor = getContentResolver().query(uri,
 									null, null, null, null);
 							if (instanceCursor.getCount() != 1) {
-								this.createErrorDialog("Bad URI: " + uri, EXIT);
+								this.createErrorDialog(getString(R.string.bad_uri) + uri, EXIT);
```

Comment:
Instead of hardcoding the order of the text and URI, the URI should be a parameter so that RTL languages can change the order.